### ISSUE TITLE
Share function/ collaboration ability for researchers

### ISSUE THREAD

**Issue by gabi-gabs** [HumanRightsWatch/VHS#23]
Multiple researchers can add videos and photos to the same Collection with a Share function between them. I don't know how best this could work in terms of permissioning and access between users.

**Comment by gabi-gabs** [comment]
This is really great - thank you. 

Could you add this to the text 'Select the people you want to share your Collection with on the left, then click on the right arrow to move them to the panel on the right. When you have finished press the blue Save button'

**Comment by gabi-gabs** [comment]
The list of available contributors- how is this list built? Is it those who have used VHS before?

**Comment by U039b** [comment]
Yes, the list of available contributors contains who has already logged-in VHS.

**Comment by gabi-gabs** [comment]
Thank you - so the workflow is anyone you want to share the videos with must already have used VHS before. I'll add this to the documentation on Gnosis
